src/ThemeProvider/ThemeProvider.test.tsx

```diff
@@ -0,0 +1,96 @@
+import { render, screen } from '@testing-library/react';
+import UIKitThemeProvider from '.';
+import { jsx, css, ThemeProvider as ProjectThemeProvider } from '@emotion/react';
+import Template from '../Template';
+import defaultTheme from '../theme/defaultTheme';
+
+// debug helper
+function getStyle(query, property) {
+  const el = document.querySelectorAll(query)[0];
+  const styles = getComputedStyle(el);
+  return styles.getPropertyValue(property);
+}
+
+const ThemesWrapper = ({ projectTheme = {}, uikitTheme = undefined, children }) => (
+  <ProjectThemeProvider theme={projectTheme}>
+    {/*@ts-ignore*/}
```

Comment:
sheer curiosity: what checks is this avoiding?

Reply:
I think this is something I ended up fixing :)
will double check and update. TS didn't like the shape of my theme prop

Reply:
🤔 yeah, the TS inference vs annotation gets finicky, doesn't it?